### PR TITLE
jsonp tck bundle generation.

### PR DESCRIPTION
**Pull request by gurunrao**
The PR creates tck bundle with structure as follows:

-   jsonp-tck/
-   jsonp-tck/LICENSE.md
-   jsonp-tck/artifacts/
-   jsonp-tck/artifacts/artifact-install.sh (script for installation of Artifacts to local maven repository.)
-   jsonp-tck/artifacts/jakarta.json-tck-2.1.0-SNAPSHOT.pom
-   jsonp-tck/artifacts/jakarta.json-tck-common-2.1.0-SNAPSHOT.jar
-   jsonp-tck/artifacts/jakarta.json-tck-common-2.1.0-SNAPSHOT.pom
-   jsonp-tck/artifacts/jakarta.json-tck-common-2.1.0-SNAPSHOT-sources.jar
-   jsonp-tck/artifacts/jakarta.json-tck-tests-2.1.0-SNAPSHOT.jar
-   jsonp-tck/artifacts/jakarta.json-tck-tests-2.1.0-SNAPSHOT.pom
-   jsonp-tck/artifacts/jakarta.json-tck-tests-2.1.0-SNAPSHOT-sources.jar
-   jsonp-tck/artifacts/jakarta.json-tck-tests-plugability-2.1.0-SNAPSHOT.jar
-   jsonp-tck/artifacts/jakarta.json-tck-tests-plugability-2.1.0-SNAPSHOT.pom
-   jsonp-tck/artifacts/jakarta.json-tck-tests-plugability-2.1.0-SNAPSHOT-sources.jar

#Steps to build the dist for EFTL bundle:
```
cd jsonp/tck/
mvn clean install -P EFTL 
```

#Steps to build the dist for EPL bundle:
```
cd jsonp/tck/
mvn clean install 
```

Signed-off-by: gurunandan.rao@oracle.com <gurunandan.rao@oracle.com>

### CODE REVIEW

File: tck/tck-dist/pom.xml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                  <id>distribution</id>
+                  <phase>package</phase>
+                  <goals>
+                     <goal>single</goal>
+                  </goals>
+                  <configuration>
+                     <descriptors>
+                        <descriptor>src/main/assembly/assembly.xml</descriptor>
+                     </descriptors>
+                     <finalName>${bundle-name}-${project.version}</finalName>
```
Comment (lukasj) [inline tck/tck-dist/pom.xml:134]: remove finalName, it's not needed as maven will still deploy the artifact based on the artifactId

File: tck/tck-dist/src/main/assembly/assembly.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lukasj) [inline tck/tck-dist/src/main/assembly/assembly.xml:1]: missing license

**Review by lukasj — COMMENTED** [review]
I'd avoid extra definitions of the source/javadoc plugins (unless needed for doclint:none) since they are inherited through oss-release profile from the parent ee4j-parent

File: tck/tck-dist/pom.xml (hunk 1 above)
Comment (gurunrao) [reply tck/tck-dist/pom.xml:134]: `<finalName>${bundle-name}-${project.version}</finalName>` will be use for naming generated TCK zip bundle(for EPL license distribution - jsonp-tck-2.1.0-SNAPSHOT.zip, EFTL license distrubution- jakarta-jsonp-tck-2.1.0-SNAPSHOT.zip)

**Comment by gurunrao** [comment]
> * SNAPSHOT

source plugin is used for generation "*-source.jar" jars. Following files are part of TCK bundle:
jsonp-tck/artifacts/jakarta.json-tck-common-2.1.0-SNAPSHOT-sources.jar
jsonp-tck/artifacts/jakarta.json-tck-tests-2.1.0-SNAPSHOT-sources.jar
jsonp-tck/artifacts/jakarta.json-tck-tests-plugability-2.1.0-SNAPSHOT-sources.jar

**Comment by gurunrao** [comment]
> I'd avoid extra definitions of the source/javadoc plugins (unless needed for doclint:none) since they are (or can be) inherited through oss-release profile from the parent ee4j-parent

source plugin is used for generation "*-source.jar" jars. Following files are part of TCK bundle:
jsonp-tck/artifacts/jakarta.json-tck-common-2.1.0-SNAPSHOT-sources.jar
jsonp-tck/artifacts/jakarta.json-tck-tests-2.1.0-SNAPSHOT-sources.jar
jsonp-tck/artifacts/jakarta.json-tck-tests-plugability-2.1.0-SNAPSHOT-sources.jar

File: tck/tck-dist/src/main/assembly/assembly.xml (hunk 2 above)
Comment (gurunrao) [reply tck/tck-dist/src/main/assembly/assembly.xml:1]: done.

**Review by lukasj — APPROVED** [review]
LGTM (inclusion of docs can be done in an extra PR)

**Pull request merged**